youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +850,22 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if entry['title']:
+                        skipnextvid = False
+                        while True:
+                            answer = compat_input('Do you want to download video %s/%s: "%s"? (y/n) ' %
+                                                  (i, n_entries, entry['title'].encode('ascii', 'ignore')))
```

Comment:
@yan12125 That won't break them. They just aren't display properly. Problem is that compat_input can't cope with non-ascii titles. I checked that.
So without encode() we're getting an exception. With encode() we won't get non-ascii chars. Any better suggestions?